engine/src/main/java/org/terasology/rendering/nui/widgets/UISlider.java

```diff
@@ -265,4 +270,12 @@ public void setPrecision(int precision) {
     private int pixelOffsetFor(float val, int width) {
         return TeraMath.floorToInt(width * (val - getMinimum()) / getRange());
     }
+
+    public interface UISliderOnChangeListener {
```

Comment:
An inner interface *and* it is followed by a regular class method? That's what you get for rushing! :-)

This even triggers a Checkstyle warning about the order in which stuff is written. Just for clarity could you entirely extract the interface instead? Sure it'll be tiny but files are cheap. Then it can get some javadoc too